### PR TITLE
Define and activate release v11.5.5 for AWS

### PR DESCRIPTION
**Pull request by alex-dabija**
AWS v11.5.5 release updates aws-cni and containerlinux.

### CODE REVIEW

**Comment by paurosello** [comment]
Updated components, how do I recreate the diff file?

**Comment by alex-dabija** [comment]
> Updated components, how do I recreate the diff file?

I updated release.diff . You can use the following command.

`diff aws/v11.5.4/release.yaml aws/v11.5.5/release.yaml -y -t > aws/v11.5.5/release.diff`

**Comment by paurosello** [comment]
We can go ahead I think

File: aws/v11.5.5/release.diff [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    version: 3.15.1                                                    version: 3.15.1
+  - name: cert-operator                                              - name: cert-operator
+    reference: 0.1.0-2                                                 reference: 0.1.0-2
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 0.1.0                                                     version: 0.1.0
+  - name: cluster-operator                                           - name: cluster-operator
+    releaseOperatorDeploy: true                                        releaseOperatorDeploy: true
+    version: 2.3.4                                                     version: 2.3.4
+  - name: containerlinux                                             - name: containerlinux
+    version: 2512.2.1                                                  version: 2512.2.1
```
Comment (alex-dabija) [inline aws/v11.5.5/release.diff:58]: The containerlinux version is still the same. Is this normal? Shouldn't it be '2512.5.0' which is the latest stable version?

**Review by paurosello — APPROVED** [review]

**Pull request merged**